src/test/CMakeLists.txt

```diff
@@ -157,10 +157,17 @@ target_link_libraries(${unit_tests_executable}
 #TODO: Do we need this?  Does not go here.
 #    -OPT:NOREF
 )
+
+# Provide a string variant of the INTERNAL_LINKER_FLAGS list
+set(flags)
+foreach(flag_item ${INTERNAL_LINKER_FLAGS})
```

Comment:
this is a bit hacky, since it has to be called in each cmake file(?) but since it works and has been a lot of work to change, let's leave it now.

Reply:
> this is a bit hacky,
I agree.  This was a last minute change to get this to work.  Unfortunately, there is no `target_linker_options()' CMake function that takes a target specific linker flags like they have for the compiler flags.  Well actually they do but it requires CMake 3.11.  There may be another function that I could use that takes a list.  If not I can go back to CommonCompilerConfig.cmake and change it to be a string since I don't use generator expressions in the link flags.